lib/cc/pull_requests.rb

```diff
@@ -2,7 +2,28 @@ class CC::PullRequests < CC::Service
   def receive_test
     setup_http
 
-    receive_test_status
+    response = receive_test_status
```

Comment:
Don't we want to do this for the PR receive, not the test coverage receive event?